CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ Classify the change according to the following categories:
     ### Deprecated
     ### Removed
 
+## load-year-align
+### Fixed
+- Align heating and cooling load profiles based on electric load year input, if using custom electric load profile with simulated (CRB or schedule-based flatloads) heating/cooling loads
+### Changed
+- Make `year` input required with `ElectricLoad.loads_kw` input
```

Comment:
Also for the custom heating and cooling load inputs, correct? Could you add here?

Reply:
Updated the line to include any load profile input.